client/src/theme/STACSS/_structure.scss

```diff
@@ -73,9 +73,12 @@
 }
 
 @mixin containerNarrow {
+  // Default max width
   max-width: $containerWidthNarrow;
   margin-right: auto;
   margin-left: auto;
+  padding-right: $containerPaddingMin;
+  padding-left: $containerPaddingMin;
```

Comment:
Name of variable `containerPaddingMin` should be written in all lowercase letters with hyphens instead of underscores